Makefile

```diff
@@ -119,6 +119,7 @@ debug_run:	fclean $(DEBUG_BINARY)
 tests_run:
 			$(MAKE) unit_tests
 			$(MAKE) func_tests
+			$(MAKE) mem_checks
```

Comment:
make sure this won‘t display compilation logs as another PR is trying to make tests_run silently